### PR TITLE
Fix: Fetching task assignment resolution states

### PR DESCRIPTION
**Pull request by pramodsum**
* Chore: Ensure pending tasks assignments have an incomplete resolution_state

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @pramodsum has signed the CLA. Thanks for the pull request!

File: src/components/ContentSidebar/ActivityFeed/task/Task.js [hunk 1]
```diff
@@ -130,6 +129,8 @@ class Task extends React.Component<Props> {
                                                     }
                                                 />
                                             );
+                                        default:
```
Comment (jeremypress) [inline src/components/ContentSidebar/ActivityFeed/task/Task.js]: Should we avoid rendering the whole task if there is no assignee? We would have to bump the check up a level.

File: src/components/ContentSidebar/ContentSidebar.js [hunk 2]
```diff
@@ -547,31 +547,6 @@ class ContentSidebar extends PureComponent<Props, State> {
...
 
-    /**
-     * Update Tasks to include task assignments
-     *
-     * @private
-     * @param {Array<TaskAssignment>} entries - Box task assignment entries
-     * @param {TaskAssignments} assignments - Box task assigments
-     * @return {TaskAssignments} Updated Box task assignments
-     */
-    populateTaskAssignments(entries: Array<TaskAssignment>, assignments: TaskAssignments): TaskAssignments {
```
Comment (jeremypress) [inline src/components/ContentSidebar/ContentSidebar.js]: Gracias

File: src/components/ContentSidebar/ActivityFeed/task/Task.js (hunk 1 above)
Comment (pramodsum) [reply src/components/ContentSidebar/ActivityFeed/task/Task.js]: I can make sure we check for that.

**Pull request merged**